Makefile

```diff
@@ -64,8 +64,15 @@ build-sol:
 	git submodule update --init --recursive
 	forge build --names --force
 
+build-vyper:
+	poetry run python vyper_contracts/scripts/compile.py
+
+install-vyper:
+	pip3 install git+https://github.com/charles-cooper/vyper@feat/eip-3074
+	vyper --version
+
 install-katana:
 	cargo install --git https://github.com/dojoengine/dojo --locked --tag v0.6.1-alpha.1 katana
 
 run-katana:
-	katana --chain-id test --validate-max-steps 16777216 --invoke-max-steps 16777216 --eth-gas-price 0 --strk-gas-price 0 --disable-fee
+	katana --chain-id test --validate-max-steps 16777216 --invoke-max-steps 16777216 --gas-price 0 --disable-fee
```

Comment:
why do you change this?

Reply:
It was not recognising --eth-gas-price and --strk-gas-price, suggested to change to --gas-price. May be I was using older version of katana. Let me upgrade my katana and try again

Reply:
It worked after installing katana from the makefile. Will revert the changes

Reply:
usually if it's not related at all with your PR, then you should not update it